torch/_dynamo/variables/builtin.py

```diff
@@ -1916,6 +1916,8 @@ def call_id(self, tx: "InstructionTranslator", *args):
         elif len(args) == 1 and isinstance(args[0], TensorVariable):
             tensor_variable = args[0]
             return tensor_variable.call_id(tx)
+        elif isinstance(args[0], variables.UserFunctionVariable):
```

Comment:
```suggestion
        elif istype(args[0], variables.UserFunctionVariable):
```

You want to exclude UserMethodVariable, etc (which will have a different ID each time).